src/eth_provider/database/types/transaction.rs

```diff
@@ -2,7 +2,16 @@ use reth_primitives::B256;
 use reth_rpc_types::Transaction;
```

Comment:
I think it's ok for now but might be worth to move all the `testing` implementation to the `test_utils` folder, such as in `test_utils/mongo/types.rs`.

Reply:
agreed let us think about this with a better structure of the mongo utils. Now that the mongo file is cleaned up, will think about that more carefully.